sbc-platform/src/pits/xlgyro_fake/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(cmake_for_gl LANGUAGES C)
```

Comment:
plz rename target, for example "xlgyro_fake_input"

Reply:
add also here cmake command to include curses library, this worked for me
project(cmake_for_gl LANGUAGES C)
find_package(Curses REQUIRED)
add_executable(cmake_for_gl main.c)
target_include_directories(cmake_for_gl PRIVATE ${CURSES_INCLUDE_DIR})
target_link_libraries(cmake_for_gl ${CURSES_LIBRARY})

Reply:
And also need to add 
`add_subdirectory(pits/xlgyro_fake)`
to sbc-platform/src/CMakeLists.txt